Source/EbsdLib/TSL/AngFields.h

```diff
@@ -78,8 +78,11 @@ class EbsdLib_EXPORT AngFields : public AbstractEbsdFields
 
 
   private:
+  public:
```

Comment:
`private: public:` should probably be `public:`
Consider deleting one of the empty lines above it.